compiler/codegen/OMRCodeGenerator.cpp

```diff
@@ -2902,20 +2891,20 @@ OMR::CodeGenerator::alignBinaryBufferCursor()
 
 
 int32_t
-OMR::CodeGenerator::setEstimatedLocationsForSnippetLabels(int32_t estimatedSnippetStart, bool isWarm)
+OMR::CodeGenerator::setEstimatedLocationsForSnippetLabels(int32_t estimatedSnippetStart)
    {
    TR::Snippet *cursor;
 
    self()->setEstimatedSnippetStart(estimatedSnippetStart);
 
    if (self()->hasTargetAddressSnippets())
       {
-      estimatedSnippetStart = self()->setEstimatedLocationsForTargetAddressSnippetLabels(estimatedSnippetStart, isWarm);
+      estimatedSnippetStart = self()->setEstimatedLocationsForTargetAddressSnippetLabels(estimatedSnippetStart);
       }
 
    for (auto iterator = _snippetList.begin(); iterator != _snippetList.end(); ++iterator)
       {
-      if ((*iterator)->isWarmSnippet() == isWarm)
+      if ((*iterator)->isWarmSnippet() == 0)
```

Comment:
Yes, but the very next PR to be submitted after this one is merged will remove all `isWarmSnippet()` calls assume they are always false (which they are).  See https://github.com/0xdaryl/omr/tree/nowarmsnippets for a sneak peak.  It has passed internal testing; I'm just waiting for this PR to be merged.